code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -562,6 +562,7 @@
 		R.use(1)
```

Comment:
Another thing to consider. R.use() can fail. Borgs can still dispense rods for this even if they are out of metal.

Reply:
seems like a borg issue more than a stack issue, but i'll see which side of it is better to fix

Reply:
It's kinda fine tbh. use can also be used to subtract larger amounts which can also fail with normal stacks.